config/prod/nextflow-vpc.yaml

```diff
@@ -0,0 +1,8 @@
+template_path: remote/vpc.yaml
+stack_name: nextflow-vpc
+parameters:
+  VpcName: nextflow-vpc
+  VpcSubnetPrefix: "10.41"
```

Comment:
Here is our list of VPCs https://sagebionetworks.jira.com/wiki/spaces/IT/pages/352976898/Sage+VPN.  

You'll just need to pick a value that isn't on the list, so how about  `172.21`?   When you decide on a value please add it to the wiki.